src/profile_manager/views.py

```diff
@@ -276,6 +280,31 @@ def get_success_url(self):
         return reverse('profiles:profile_update', args=[self.get_instance().profile.pk])
 
 
+@non_public_only_view
+@login_required
+def profile_resend_email_verification(request):
+
+    user = request.user
+    email_address = EmailAddress.objects.filter(email=user.email).first()
+
+    if not email_address or not user.email:
+        messages.error(request, "User does not have an email")
+        return redirect_to_previous_page(request)
+
+    if email_address.verified:
```

Comment:
Under what circumstances would this occur?  Maybe a comment to explain.

Reply:
Will add a comment. But this would only happen if they happen to have access to the resend verification url or click it twice